tutorials/dapt-curation/README.md

```diff
@@ -0,0 +1,67 @@
+# Data curation for DAPT (Domain Adaptive Pre-Training)
+
+[ChipNeMo](https://arxiv.org/pdf/2311.00176) is a chip design domain adapted LLM. LLama models are continually pre-trained with 20B plus tokens on domain-specific chip design data, including code, documents, etc., based on NeMo foundation models and then fine-tuned with instruction datasets from design data as well as external sources.
+
+Here, we share a tutotial with best practices on data curation for DAPT (domain-adaptive pre-training) for a ChipNeMo-like code generation use case.
+
+* In this tutorial, we will leverage chip domain/hardware datasets from open-source GitHub repositories (`.code/sources/github_repos.jsonl`), wiki URLs (`.code/sources/wikipedia_urls.jsonl`), and academic papers(`./sources/arxiv_urls.jsonl`).
+
+* `\code\data` contains curated data after processing
+
+* Notebook for data curation with NeMo Curator: `\code\DAPT_data_curation.ipynb`
```

Comment:
I'm a little confused about this: IIRC, the notebook version had issues, so we decided to publish this in the script format (`main.py`). If my understanding is correct, what's the reason for keeping the notebook?

If this is for content only, we can move the content to the blog.

Reply:
@Maghoumi The notebook is just for content and quick tests. It will also help with any demos as we want to walk through the code. Should we still remove it?

Reply:
Yes. I suggest keeping a copy of it in a branch or something for testing.
Given that this repo is production-grade, we're not allowed to push experimental/debugging content to it.

Reply:
Ok I added it through .gitignore. Will this work?